tests/e2e/lib.sh

```diff
@@ -665,6 +665,83 @@ check_for_stackrox_OOMs() {
     fi
 }
 
+declare -A POD_CONTAINERS_MAP
+fill_pod_containers_map() {
+    POD_CONTAINERS_MAP=()
+
+    POD_CONTAINERS_MAP["pod: central - container: central"]="central-[A-Za-z0-9]+-[A-Za-z0-9]+-central-previous.log"
```

Comment:
I was thinking about this also. Another aspect is that we have different pods for different test runs. (i.e. scanner-v4 runs only on scanner-v4 tests, operator runs only on openshift tests, etc.)

My thought was to **update the list when we face failure**.

Right now, tickets for "known" containers will be:
```
Pod Restarts pod: collector - container: node-inventory
```
Where for "unknown" it will be:
```
Pod Restarts collector
```
We should simply update our list if we see such tickets.